sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_directory_client.py

```diff
@@ -694,12 +701,13 @@ def get_paths(
         command = functools.partial(
             client.file_system.list_paths,
             path=self.path_name,
+            begin_from=begin_from,
```

Comment:
Not sure if you did this in Blob but I don't think this needs to be passed here since it's passed below? Looks at name_start_with in Blob.

However, you may be able to JUST pass it here and then not need to pass it below and mess with PathPropertiesPaged. I think it just needs to be passed here or in PathPropertiesPaged and passing here is certainly easier so you could try that. Tbh, I'm not entirely sure it will work though.

Sorry if I missed this in Blob but whatever we come up with here, we should go back and change in Blob as well.